apps/transport/test/transport_web/controllers/aoms_controller_test.exs

```diff
@@ -0,0 +1,129 @@
+defmodule TransportWeb.AomsControllerTest do
+  use TransportWeb.ConnCase, async: false
```

Comment:
Je préfère les passer en `async: false`, au moins je comprends pourquoi ça ne se marche pas sur les pieds avec les autres tests. J'ai creusé cette histoire de dead lock, mais je n'ai pas compris d'où ça venait. J'ai l'impression que ce sont les tests avec des aoms qui ont des parent_dataset qui posent problème.

J'ai quand même au passage compris que des tests async qui inserent des données similaires  dans des tables qui ont des contraintes d'unicité se retrouvent de fait à tourner en sync à cause des locks de la base de données. Je pense qu'on a typiquement ce cas avec tout ce qui utilise `TransportWeb.DatabaseCase`, parce que le nom des région doit être unique, mais on insert toujours les mêmes noms de régions avec de module. J'ai l'impression que les tests qui l'utilisent sont donc in fine contraints de tourner en synchrone.

Reply:
Merci beaucoup d'avoir aussi pris le temps de regarder @AntoineAugusti